plugins/generator-1.20.6/neoforge-1.20.6/templates/json/advancement.json.ftl

```diff
@@ -11,7 +11,8 @@
               </#if>
           </#if>
           "icon": {
-            ${mappedMCItemToItemObjectJSON(data.achievementIcon)}
+            ${mappedMCItemToItemObjectJSON(data.achievementIcon, "id")},
+            "count": 1

```

Comment:
Misode does not show the need for this change:

![image](https://github.com/MCreator/MCreator/assets/16374228/e60d804b-c39d-41eb-9e9d-dbb728d9c531)

Reply:
Without this change the test advancements did not show in the advancements dialog. I also noticed this change of format in vanilla advancement files.

Reply:
SUS, I guess misode is missing out this then

Reply:
Did you mean the `"count"` parameter specifically? The game still works without it even despite advancements in MC use it.

Reply:
Yes. I meant that I don't think the change was needed. If it is not needed, I would leave it out

Reply:
:heavy_check_mark: Fixed